packages/core/src/rest/ZosmfRestClient.ts

```diff
@@ -20,7 +20,6 @@ import { ZosmfHeaders } from "./ZosmfHeaders";
  * @extends {RestClient}
  */
 export class ZosmfRestClient extends RestClient {
-
     /**
```

Comment:
Could you please add the blank line back above? Sorry for being picky 😢 If Lerna sees any Git changes in the "core" package, it will require a changelog update and publish a new version of the core SDK package.